diagnostic.md

```diff
@@ -71,13 +75,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
+planning is a relative path because it does not start with a /
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+relative path - so you can move the directory around and it will alway find the image
```

Comment:
Correct, but more specifically the image file lives inside our current working directory, project ; as a result, an absolute path is not needed to access it.